code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -80,3 +85,44 @@
 		new /obj/item/stack/cable_coil(src,30,color)
 		new /obj/item/weapon/wirecutters(src)
 		new /obj/item/device/multitool(src)
+
+
+/obj/item/weapon/storage/toolbox/proc/update_force()
+	force = initial(force)
+	for (var/obj/item/I in contents)
+		force += I.w_class*1.3
+
+/obj/item/weapon/storage/toolbox/handle_item_insertion(obj/item/W as obj, prevent_warning = 0)
+	if (..(W, prevent_warning))
+		update_force()
+
+
+/obj/item/weapon/storage/toolbox/attack(mob/living/M as mob, mob/user as mob)
+	stunhit = 0
+	update_force()
+	world << "Force is [force] and shield is [M.isHoldingShield()]"
+	if (force > 14 && !M.isHoldingShield())
+		var/odds = 80

```

Comment:
I feel like this is too explicit for a stunhit. How about tying them together like `var/odds = force * 5` and also instead of `if (force > 14)` possibly `if (prob(force * 8))`

I don't believe in something suddenly being able to stun because you surpassed a magic number is all